### PR TITLE
Make Zalesak an example and a reg test

### PR DESCRIPTION
**Pull request by mbkuhn**

### CODE REVIEW

File: test/test_files/zalesak/zalesak.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+exawind:
+  nalu_wind_inp:
+    - zalesak-nalu.yaml
+  amr_wind_inp: zalesak-amr.inp
+  num_timesteps: 60
```
Comment (psakievich) [inline test/test_files/zalesak/zalesak.yaml:6]: How fast does this thing run? 60 timesteps seems like a lot.  If it takes longer than 10 seconds I'd say dial this back for the repo.

File: test/test_files/zalesak/zalesak-nalu.yaml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        - consistent_mass_matrix_png:
+            pressure: no
+      # Pressure is not fixed anywhere on the boundaries, so set it at
+      # the node closest to the specified location.
+      fix_pressure_at_node:
+        value: 0.0
+        node_lookup_type: spatial_location
+        location: [0.0, 0.25, 0.25]
+        search_target_part: [block_1]
+        search_method: stk_kdtree
```
Comment (psakievich) [inline test/test_files/zalesak/zalesak-nalu.yaml:89]:
```suggestion
```
Don't need this.

File: test/test_files/zalesak/zalesak.yaml (hunk 1 above)
Comment (mbkuhn) [reply test/test_files/zalesak/zalesak.yaml:6]: 10.28 sec locally. The iterations don't take long because there are no pressure solves

**Review by psakievich — APPROVED** [review]

**Comment by marchdf** [comment]
Any reason why we haven't merged this? It looks all set?

**Pull request merged**